book/src/leader-validator-transition.md

```diff
@@ -0,0 +1,85 @@
+# Leader-to-Validator Transition
+
+A fullnode typically operates as a validator. If, however, a staker delegates
+its stake to a fullnode, it will occasionally be selected as a *slot leader*.
+As a slot leader, the fullnode is responsible for producing blocks during an
+assigned *slot*. A slot has a duration of some number of preconfigured *ticks*.
+The duration of those ticks are estimated with a *PoH Recorder* described later
+in this document.
+
+## BankFork
+
+BankFork tracks changes to the bank state over a specific slot.  Once the final
+tick has been registered the state is frozen. Any attempts to write to are
+rejected.
+
+## Validator
+
+A validator operates on many different concurrent forks of the bank state until
+it can prove a PoH record at a height within the slot it is scheduled to be the
```

Comment:
This sentence reads weird to me, I get lost during the "until it can prove..." part.

Reply:
How about? "A validator operates on many different concurrent forks of the bank state until
it generates a PoH hash with a height within its leader slot."